src/transformers/testing_utils.py

```diff
@@ -233,6 +234,32 @@ def require_faiss(test_case):
         return test_case
 
 
+def require_optuna(test_case):
```

Comment:
Note that we don't have any setup installing optuna or ray on the CI so for now, any tests marked by those decorators is just never going to be executed.

Reply:
OK.

Reply:
Is there something I can do to add optuna / ray test cases on the CI with the proper configuration ?

Reply:
OK, so I will have a look how to integrate this.